src/deepcell/data_splitter.py

```diff
@@ -63,12 +70,46 @@ def __init__(self, model_inputs: List[ModelInput], train_transform=None,
         self._use_correlation_projection = use_correlation_projection
         self._centroid_brightness_quantile = centroid_brightness_quantile
         self._centroid_use_mask = centroid_use_mask
+        self._n_depth_bins = n_depth_bins
 
         if center_roi_centroid not in ('test', True, False):
             raise ValueError(f'Invalid value for center_soma. Valid '
                              f'values are "test", True, or False')
         self._center_roi_centroid = center_roi_centroid
 
+        self._create_experiment_labels(experiment_metadatas)
+
+    def _create_experiment_labels(
+            self,
+            experiment_metadatas: List[ExperimentMetadata]):
+        """
+        """
+        self._exp_labels = np.full(len(experiment_metadatas), -1, dtype=int)
```

Comment:
Instead of setting `exp_labels` as an instance attribute, can you call this method from `get_train_test_split`? IMO this logic is local to getting the train/test split.